mappings/net/minecraft/block/entity/EndGatewayBlockEntity.mapping

```diff
@@ -42,17 +43,23 @@ CLASS net/minecraft/class_2643 net/minecraft/block/entity/EndGatewayBlockEntity
 	METHOD method_11421 needsCooldownBeforeTeleporting ()Z
 	METHOD method_30276 canTeleport (Lnet/minecraft/class_1297;)Z
 		ARG 0 entity
-	METHOD method_31698 (Lnet/minecraft/class_3218;Lnet/minecraft/class_243;)Z
+	METHOD method_31698 isChunkEmptyAt (Lnet/minecraft/class_3218;Lnet/minecraft/class_243;)Z
 		ARG 0 world
-	METHOD method_31699 (Lnet/minecraft/class_3218;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2338;
+		ARG 1 pos
+	METHOD method_31699 setupExitPortalLocation (Lnet/minecraft/class_3218;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2338;
+		COMMENT Finds teleport location and creates an island to teleport to (if there is none).
+		COMMENT
+		COMMENT <p>This does not create an exit portal.</p>
+		COMMENT
+		COMMENT @return the position of the exit portal
 		ARG 0 world
 		ARG 1 pos
 	METHOD method_31700 clientTick (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;Lnet/minecraft/class_2643;)V
 		ARG 0 world
 		ARG 1 pos
 		ARG 2 state
 		ARG 3 blockEntity
-	METHOD method_31701 (Lnet/minecraft/class_3218;Lnet/minecraft/class_2338;)Lnet/minecraft/class_243;
+	METHOD method_31701 findTeleportLocation (Lnet/minecraft/class_3218;Lnet/minecraft/class_2338;)Lnet/minecraft/class_243;
```

Comment:
```suggestion
	METHOD method_31701 findTeleportPosition (Lnet/minecraft/class_3218;Lnet/minecraft/class_2338;)Lnet/minecraft/class_243;
```

Reply:
How is a "location" more accurate than a "position", I wonder?